py/desitarget/mock/build.py

```diff
@@ -1391,7 +1394,7 @@ def join_targets_truth(mockdir, outdir=None, overwrite=False, comm=None):
         todo = comm.bcast(todo, root=0)
 
     if todo['sky']:
-        _merge_file_tables(mockdir+'/*/*/sky-*.fits', 'SKY_TARGETS',
+        _merge_file_tables(mockdir+'/*/*/sky-*.fits', 'SKY',
                            outfile=outdir+'/sky.fits', comm=comm,
                            overwrite=overwrite)
                            #addcols=dict(OBSCONDITIONS=obsmask.mask('DARK|GRAY|BRIGHT')))
```

Comment:
@sbailey FYI: I called this extension `SKY_TARGETS` to match the name of the extension written out by `io.write_skies` for "real" data.

Reply:
Thanks.  I incorrectly changed that when my minitest notebook was accidentally picking up desitarget master instead of this branch, and files were created with the old SKY name and then I was using this branch to try to merge them.  I reverted back to SKY_TARGETS.

I've made it past mock targets with this branch and am now hitting some fiberassign bookkeeping issues.  All my changes are pushed to github, but I'd like to resolve whether the fiberassign problems are on the fiberassign side or on the mock inputs side or user error before merging this.  I hope to get back to this later today after juggling some other flaming cats.